src/firewall/core/io/policy.py

```diff
@@ -77,6 +77,19 @@ def common_startElement(obj, name, attrs):
             else:
                 log.warning("Protocol '%s' already set, ignoring.",
                             attrs["value"])
+    
+    elif name == "tcp-mss-clamp":
+        if obj._rule:
+            if obj._rule.element:            
+                log.warning("Invalid rule: More than one element in rule '%s', ignoring.",
+                            str(obj._rule))
+                obj._rule_error = True
+                return True
+            obj._rule.element = rich.Rich_Tcp_Mss_Clamp(attrs["value"])
+        else:
+            log.warning("Tcp-mss-clamp value '%s' already set, ignoring.",
+                        attrs["value"])
```

Comment:
This is non-sense. The entire `else` block in invalid because we should never encounter the `tcp-mss-clamp` element outside of rich rules. I think you can simply remove the entire `else` block.. or better yet throw an error with a message "tcp-mss-clamp is only valid inside of rich elements."